client/jetpack-connect/auth-logged-in-form.jsx

```diff
@@ -137,7 +135,7 @@ class LoggedInForm extends Component {
 
 		// For SSO, WooCommerce Services, and JPO users, do not display plans page
 		// Instead, redirect back to admin as soon as we're connected
-		if ( nextProps.isSSO || nextProps.isWoo || this.isFromJpo( nextProps ) ) {
+		if ( nextProps.isSso( nextProps ) || this.isWoo( nextProps ) || this.isFromJpo( nextProps ) ) {
```

Comment:
Should this be `this.isSso( nextProps )` instead of `nextProps.Sso( nextProps )`?

Reply:
Yes!

Reply:
Fixed in 38112cf107c9d7cfe33a338aadab532c61015519, thanks!